client/src/components/ElementEditor/Element.js

```diff
@@ -39,32 +39,19 @@ const Element = (props) => {
   const [doPublishElementAfterSave, setDoPublishElementAfterSave] = useState(false);
   const [ensureFormRendered, setEnsureFormRendered] = useState(false);
   const [formHasRendered, setFormHasRendered] = useState(false);
-  const [doDispatchAddFormChanged, setDoDispatchAddFormChanged] = useState(false);
-  const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false);
```

Comment:
No need to separately keep track of `hasUnsavedChanges` because we can just rely on `formDirty` instead.
No need to use `doDispatchAddFormChanged` because redux form keeps track of that for us automatically.